scala/spark-pubsub/README.md

```diff
@@ -0,0 +1,161 @@
+# Example: Integrating Spark Streaming with Cloud Pubsub
+
+This example uses [Spark Streaming][spark-streaming] to pull for new files
+every 30 seconds and perform a simple Spark job that counts the number of
+times a word appears in each new file. The Spark job uses
+[Cloud Bigtable][landing-page] to store the results.
+
+[spark-streaming]: https://spark.apache.org/
+[landing-page]: https://cloud.google.com/bigtable/docs/
+
+Please note that we encourage users to develop programs on their local 
+machine, not on the VMs. User will develop a Spark application on his/her 
+computer, use sbt to build (explained in the “Building the code sample” 
+section below), then transfer the application jar to a GCE VM. The GCE 
+VMs can be configured with bdutil to run Spark applications (link to 
+“Running the code sample with spark-submit”).
+
+
+## Before you start
+
+Before you run this code sample, you'll need to complete the following tasks:
+
+1. [Create a Cloud Bigtable cluster][create-cluster]. Be sure to note the
```

Comment:
I was able to open the links on my computer by clicking "Create a Cloud Bigtable cluster" from here https://github.com/taragu/cloud-bigtable-examples/blob/spark/scala/spark-pubsub/README.md. Do I still need to change it to [text](link)?